src/modules/QtQml/Timer.js

```diff
@@ -27,14 +27,21 @@ registerQmlType({
         }
     }
 
-    this.start = function() {
-        if (!this.running) {
-            this.running = true;
-            prevTrigger = (new Date).getTime();
+    /* This ensures that if the user toggles the "running" property manually,
+     * the timer will trigger. */
+    this.runningChanged.connect(function() {
+        if (self.running) {
+            prevTrigger = new Date().getTime();
             if (this.triggeredOnStart) {
```

Comment:
Should be good now. I just used the `.connect(this, function() {` method, as it is seems to be more common in the current implementation.